en/components/themes/typography.md

```diff
@@ -6,149 +6,210 @@ _extraFont: https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,70
-            non ipsum tempus, et sodales diam luctus. Nunc sagittis lectus id risus hendrerit commodo. Donec eget
-            lobortis velit. Pellentesque sollicitudin eros sit amet nulla ornare, vitae maximus nisi elementum. Cras
-            eleifend ante at lectus molestie, in sagittis nisl cursus. Vestibulum eu convallis nibh. Proin viverra
-            pellentesque metus, quis porta massa mollis id. Integer consectetur nunc ut lobortis egestas. Curabitur
-            viverra nisi lectus, vel feugiat massa mattis ac. Ut viverra pretium varius. Sed pharetra tempus elit
-            ut lacinia. Donec nisi mi, imperdiet sed finibus ac, aliquet sed dolor.
-        </p>
-        <div class="spec">
-            <span>Font size: <b>1rem</b> (16px)</span>
-            <span>Line height: <b>1.6875rem</b> (27px)</span>
-            <span>Margin top: <b>0</b></span>
-            <span>Margin bottom: <b>1.6875rem</b> (27px)</span>
-        </div>
-    </div>
-</article>
+We have selected [Titillium Web](https://fonts.google.com/selection?selection.family=Titillium+Web:300,400,600,700) to be the default font for Ignite UI for Angular. To use it you have to host it yourself, or include it from Google Fonts:
 
-<div class="divider"></div>
+```html
+<link href="https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,700" rel="stylesheet">
+```
+
+There are a several mixins and functions that are used to set and retrieve category styles to/from a type scale. Those are:
+
+- `igx-type-style` [function] - Returns a set of style rules to be used by a type scale category.
+- `igx-type-scale` [function] - Returns a set of 13 style categories.
+- `igx-type-scale-category` [function] - Returns a map of style rules from a type scale and category.
+- `igx-type-style` [mixin] - Adds style rules to a selector from a specific type scale and category.
+- `igx-typography` [mixin] - Defines the global application typography styles.
+
+
+Let's take a close look at what each one of the aforementioned mixins and functions do.
+
+#### The Type Style
+The `igx-type-style` function is an interface-like function that simply ensures that certain arguments are passed as part of the style set for a scale category. Say, for instance, that we want to define a new set of style rules for the `h1` scale category. To do so, we would simply write:
+
+```scss
+$h1-style: igx-type-style(
+    $font-size: 112px,
+    $font-weight: 600,
+    $line-height: 96px
+);
+```
+
+> [!NOTE]
+> Any properties that you do not pass, such as `$font-family`, `letter-spacing`, etc. will be automatically replaced with the default values as specified in the `$default-type-scale` for the category you want to use your style for.
+
+
+#### The Type Scale
+
+The type scale is a map of type styles for all 13 scale categories. To generate a new type map all you have to do is write.
```

Comment:
To generate a new type map, all you have to do is write the following SCSS:
"SCSS" here is changeable.  I don't know what the best word would be.  "rule"?  "style"?